server.js

```diff
@@ -0,0 +1,31 @@
+import "babel-polyfill";
+import koa from "koa";
+import webpackDevMiddleware from "koa-webpack";
+import webpackDevConfig  from "./config/webpack.config.dev.js";
+import React from 'react';
+import serve from 'koa-static';
+
+import path from 'path';
+
+//import {handleRender} from './render-react-app.jsx';
+
+const app = new koa();
+
+console.log(`running app in ${app.env} mode`);
+
+app.use(serve(`${__dirname}/public`));
+//app.use(handleRender);
+
+//function renderFullPage(html, preloadedState) { /* ... */ }
+
+if (app.env !== 'prod') {
+    console.log('using webpack-dev-middleware');
+
+    app.use(webpackDevMiddleware({
```

Comment:
this middleware used in development mode to serve bundled client code from memory (We could use webpack-dev-server as an alternative but it would mean running two node servers simultaniously)